addons/l10n_latam_check/models/account_chart_template.py

```diff
@@ -0,0 +1,36 @@
+from odoo import models, _
+import logging
+_logger = logging.getLogger(__name__)
+THIRD_CHECKS_COUNTRY_CODES = ["AR"]
```

Comment:
HI william
The idea is not to make inheritable but to make it more explicit and clear how to extend the module so that it can be used on other localizations.
Actually the module can be used, the difference is that this journals are created by default or not.

Maybe the name of the variable could be different?

For me no problem to add a method or whatever odoo believes it's better

cc @jco-odoo

Reply:
Done, method added